profiles/views_test.py

```diff
@@ -64,6 +65,27 @@ def setUp(self):
             )
         self.url2 = reverse('profile-detail', kwargs={'user': username})
 
+        # Monkey patch get_serializer_class so its return value can be asserted. This is reverted in tearDown
```

Comment:
`self.returned_serializer` is asserted in a couple places, for example https://github.com/mitodl/micromasters/pull/956/files/cef11cc3a8a70363495f85b838ac26e0d3cce0d1#diff-fe503725d8f59174a69abe2613648b44R326